zerver/lib/string_validation.py

```diff
@@ -16,12 +16,19 @@
 }
 
 
+def is_character_printable(char: str) -> bool:
+    unicode_character = unicodedata.category(char)
```

Comment:
This should be called `unicode_category`; it's not a character.

Reply:
Ah, thanks, didn't notice that when refactoring